meta/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+galaxy_info:
+  author: Caktus Consulting Group, LLC
+  description: Django project deployment -- rabbitmq
+  company: Caktus Consulting Group, LLC
+  license: BSD
+  min_ansible_version: 1.9
```

Comment:
There's really not much difference between the versions that I've noticed, so my preference is to avoid locking out potential 3rd party users of our roles from using a version that's only a little bit behind.